examples/howto/marginalizing-models.ipynb

```diff
@@ -0,0 +1,1440 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    
</pre><p>A bit shorter, and I would put it inside the model definition (optional)?</p><p> </p><p><code>mu = pt.as_tensor([-3.0, 3.0])</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/628/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>